src/transport/raw/TCP.cpp

```diff
@@ -168,37 +168,21 @@ TCPBase::ActiveConnectionState * TCPBase::FindActiveConnection(const Inet::TCPEn
     return nullptr;
 }
 
-CHIP_ERROR TCPBase::SendMessage(const PacketHeader & header, const Transport::PeerAddress & address,
-                                System::PacketBufferHandle msgBuf)
+CHIP_ERROR TCPBase::SendMessage(const Transport::PeerAddress & address, System::PacketBufferHandle msgBuf)
 {
-    // Sent buffer data format is:
-    //    - packet size as a uint16_t (inludes size of header and actual data)
-    //    - header
-    //    - actual data
-    const size_t prefixSize = header.EncodeSizeBytes() + kPacketSizeBytes;
-
     VerifyOrReturnError(address.GetTransportType() == Type::kTcp, CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
Please keep the documentation about why we are doing the games with kPacketSizeBytes.

```suggestion
    // Sent buffer data format is:
    //    - packet size as a uint16_t
    //    - actual data
    
    VerifyOrReturnError(address.GetTransportType() == Type::kTcp, CHIP_ERROR_INVALID_ARGUMENT);
```